pyuploadcare/api_resources.py

```diff
@@ -231,24 +231,76 @@ def store(self):
         """
         self._info_cache = rest_request('PUT', self._api_storage_uri)
 
-    def copy(self, effects=None, target=None):
-        """Creates File copy
+    def local_copy(self, effects=None, store=None):
+        """Creates a Local File Copy on Uploadcare Storage
 
-        If ``target`` is ``None``, copy file to Uploadcare storage otherwise
-        copy to target associated with project.
-        Add ``effects`` to ``self.default_effects`` if any.
+        - effects:
+            Adds CDN image effects to ``self.default_effects`` if any;
+        - store:
+            If ``store`` option is set to False the copy of your file will be deleted
+            in 24 hour period after the upload.
```

Comment:
Changed wording. Actual description is  given in definition of cdn_url function.